src/client/components/EmailRegistration/EmailForm.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import './EmailForm.css';
+
+export default function EmailRegistration({ formWidth, fontFamily }) {
+  const [email, setEmail] = useState('');
+  const [error, setError] = useState('');
+  // onchange function for input
+  const handleEmailInput = (e) => {
+    setEmail(e.target.value);
+    setError('');
+  };
+  // onSubmit function
+  const handleSubmit = (e) => {
+    const pattern = new RegExp(
+      /^(("[\w-\s]+")|([\w-]+(?:\.[\w-]+)*)|("[\w-\s]+")([\w-]+(?:\.[\w-]+)*))(@((?:[\w-]+\.)*\w[\w-]{0,66})\.([a-z]{2,6}(?:\.[a-z]{2})?)$)|(@\[?((25[0-5]\.|2[0-4][0-9]\.|1[0-9]{2}\.|[0-9]{1,2}\.))((25[0-5]|2[0-4][0-9]|1[0-9]{2}|[0-9]{1,2})\.){2}(25[0-5]|2[0-4][0-9]|1[0-9]{2}|[0-9]{1,2})\]?$)/i,
+    );
+    //  if enter key is pressed, validating user email.
+    if (e.key === 'Enter') {
+      e.preventDefault();
+      if (email === '') {
+        setError('please enter your email');
+      } else if (!pattern.test(email)) {
+        setError("sorry,this email can't be registered.Let's try another one.");
+      }
+    }
+  };
+
+  return (
+    <form
+      className="email_form"
+      style={{
+        width: `${formWidth}px`,
+        fontFamily: `${fontFamily}`,
```

Comment:
is there a reason to pass the fontfamily? will it ever change dynamically?

Reply:
@bytewiz, Thank You for your review...!, In the issue 66, they mentioned to use knobs/controls for font family, so i gave that dynamicness, but really not sure not about whether should i do that or not...?